content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -115,6 +141,54 @@ backend is typically a configuration option of the [Ingress controller](/docs/co
 If none of the hosts or paths match the HTTP request in the Ingress objects, the traffic is
 routed to your default backend.
 
+## Ingress Class
+
+Ingresses can be implemented by different controllers, often with different
+configuration. Each Ingress should specify a class, a reference to an
+IngressClass resource that contains additional configuration including the name
+of the controller that should implement the class.
+
+```yaml
+apiVersion: networking.k8s.io/v1beta1
+kind: IngressClass
+metadata:
+  name: external-lb
+spec:
+  controller: example.com/ingress-controller
+  parameters:
+    apiGroup: k8s.example.com/v1alpha
+    kind: IngressParameters
+    name: external-lb
+```
+
+IngressClass resources contain an optional parameters field. This can be used to
+reference additional configuration for this class.
+
+### Deprecated Annotation
+
+Before the IngressClass resource and ingressClassName field were added in
+Kubernetes 1.18, Ingress classes were specified with a
+`kubernetes.io/ingress.class` annotation on the Ingress. This annotation was
+never formally defined, but was widely supported by Ingress Controllers.
+
+The newer `ingressClassName` field on Ingresses is a replacement for that
+annotation, but is not a direct equivalent. While the annotation was generally
+used to reference the name of the Ingress controller that should implement the
+Ingress, the field is a reference to an IngressClass resource that contains
+additional Ingress configuration, including the name of the Ingress controller.
+
+### Default Ingress Class
+
+An IngressClass can be marked as default within a Kubernetes cluster. Setting
+the `ingressclass.kubernetes.io/is-default-class` annotation to `true` on an
+IngressClass resource will ensure that new Ingresses without an `ingressClassName`
+field specified will be assigned this default IngressClass.
+
+Note: If more than one IngressClass is concurrently marked as default in a
+cluster, new Ingresses without the `ingressClassName` field specified will not
+be able to be created. This can be resolved by ensuring that at most 1
+IngressClasess are marked as default in the cluster.
```

Comment:
```suggestion
{{< caution >}}
If you have more than one IngressClass marked as the default for your
cluster, the API server prevents creating new Ingress objects that don't
have an `ingressClassName` specified. You can resolve this by ensuring
that at most 1 IngressClasess are marked as default in your cluster.
{{< /caution >}}
```
BTW what kind of rejection? I guessed. It's best to be clear.

Reply:
It's actually rejected by the corresponding admission controller. Wasn't sure if I should put the name since it's really just the "DefaultIngressClass" admission controller. Updated to use that formatting + reference the admission controller.

Reply:
Ah that makes sense. Naming the thing both helps avoid writing in the passive vice and informs readers who might want to know the detail.